### PR TITLE
[Console-IoT]-Add ID for endpoint components

### PR DESCRIPTION
**Pull request by suyash-naithani**
### Type of change

- Refactoring

### Description

Add meaningful ID for endpoint components

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/modules/endpoints/EndpointPage.tsx [hunk 1]
```diff
@@ -68,7 +69,10 @@ export default function AddressPage() {
     }
   };
   return (
-    <PageSection variant={PageSectionVariants.light}>
+    <PageSection
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/endpoints/EndpointPage.tsx:72]: Page section doesn't need id

File: console/console-init/ui/src/modules/endpoints/EndpointPage.tsx [hunk 2]
```diff
@@ -59,6 +59,7 @@ export default function AddressPage() {
     if (data && data.total > 10) {
       return (
         <TablePagination
+          id="endpoint-page-pagination"
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/endpoints/EndpointPage.tsx:62]: The `TablePagination` component does not seem to handle ids in its definition.

File: console/console-init/ui/src/modules/endpoints/EndpointPage.tsx (hunk 1 above)
Comment (drcoolsanjeev) [reply console/console-init/ui/src/modules/endpoints/EndpointPage.tsx:72]: It's not fixed.

File: console/console-init/ui/src/modules/endpoints/EndpointPage.tsx (hunk 2 above)
Comment (suyash-naithani) [reply console/console-init/ui/src/modules/endpoints/EndpointPage.tsx:62]: As resolved in chat :)

File: console/console-init/ui/src/modules/endpoints/EndpointPage.tsx (hunk 1 above)
Comment (suyash-naithani) [reply console/console-init/ui/src/modules/endpoints/EndpointPage.tsx:72]: Will be removing page section id in any future file chages

**Comment by suyash-naithani** [comment]
Resolved all comments. Looks good to merge

**Pull request merged**